lang/ilias_en.lang

```diff
@@ -17089,11 +17089,11 @@ trac#:#user_total#:#User Total
 trac#:#view_mode#:#View Mode
 tstv#:#tstv_create#:#Create Test Certificate
 tstv#:#tstv_create_info#:#Select a completed test to generate a certificate for it.
-ui#:#datatable_multiactionmodal_actionlabel#:#operation
+ui#:#datatable_multiactionmodal_actionlabel#:#Action for all entries
```

Comment:
Action for All Entries

Reply:
Sorry, I have overlooked this comment. Is adjusted.